query-engine/query-engine/src/server/mod.rs

```diff
@@ -101,11 +107,7 @@ async fn graphql_handler(mut req: Request<State>) -> tide::Result {
 ///
 /// In production exposing the playground is equivalent to exposing the database
 /// on a port. This should never be enabled on production servers.
-async fn playground_handler(req: Request<State>) -> tide::Result {
-    if !req.state().enable_playground {
-        return Ok(Response::new(StatusCode::NotFound));
-    }
-
+async fn playground_handler(_: Request<State>) -> tide::Result {
```

Comment:
The playground seems to be always enabled with that change now.

Reply:
Ooph -- yep, I made the wrong route conditional. Fixed it!